projects/wave-core-new/src/lib/datasets/dataset.model.ts

```diff
@@ -0,0 +1,34 @@
+import {UUID} from '../backend/backend.model';
+import {SpatialReference} from '../operators/spatial-reference.model';
+
+export interface DataSet {
+    id: InternalDataSetId; // TODO: support all Id types
+    name: string;
+    description: string;
+    result_descriptor: ResultDescriptor;
+    source_operator: string;
+}
+
+export interface InternalDataSetId {
+    Internal: UUID;
+}
+
+export interface ResultDescriptor {
+    Vector?: {
+        data_type: 'MultiPoint' | 'MultiLineString' | 'MultiPolygon';
```

Comment:
I guess you missed data collections

Reply:
51f0fd1a6ee433c90556404a06b00f509760a077